.github/workflows/dotnet.yml

```
@@ -0,0 +1,73 @@
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
+
+      - name: Install ABP CLI
+        run: cd easyshifthq
+          dotnet tool install -g Volo.Abp.Cli
+        shell: bash
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)